src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -10382,9 +10389,17 @@ namespace HVACUnitarySystem {
 						}
 					}
 				} else {
-					if ( HeatSpeedNum <= 1 ) {
+					if ( HeatSpeedNum < 1 ) {
 						CompOffMassFlow = UnitarySystem( UnitarySysNum ).IdleMassFlowRate;
 						CompOffFlowRatio = UnitarySystem( UnitarySysNum ).IdleSpeedRatio;
+					} else if ( HeatSpeedNum == 1 ) {
+						if ( UnitarySystem( UnitarySysNum ).MultiSpeedHeatingCoil ) {
+							CompOffMassFlow = 0.0;
+							CompOffFlowRatio = 0.0;
+						} else {
+							CompOffMassFlow = UnitarySystem( UnitarySysNum ).HeatMassFlowRate( HeatSpeedNum );
+							CompOffFlowRatio = UnitarySystem( UnitarySysNum ).HeatMassFlowRate( HeatSpeedNum );
+						}
```

Comment:
This change is extremely subtle. CompOn/OffMassFlow is used in SetAverageFlowRate to set the node flow based on PLR. Yet in the MS coils, MSHPMassFlowLow is used to calculate a flow ratio wrt max flow at the operating speed, which is then used to calculate a capacity as a function of flow multiplier. These variables have to be set carefully at speed = 1. At speed > 1, High and Low are just flow at speed N and N-1. At speed = 1, for multispeed coils, the MSHPLow variable is proportional to PLR. So here, for multispeed coils, the off flow must be set to 0 so that SetAverageFlow and put the correct mass flow rate on the node. Below, MSHPLow is set to tell the coil what the flow rate is. Pretty much a hack in the original design.